orchestra/actions/install.py

```diff
@@ -165,7 +165,15 @@ def _fetch_binary_archive(self):
         binary_archive_path = pathlib.Path(binary_archive_path)
         binary_archive_root = get_worktree_root(binary_archive_path)
         binary_archive_relative_path = binary_archive_path.relative_to(binary_archive_root)
-        lfs.fetch(binary_archive_root, include=[binary_archive_relative_path])
+        failures = 0
+        while True:
+            try:
+                lfs.fetch(binary_archive_root, include=[binary_archive_relative_path])
+                break
+            except Exception as e:
```

Comment:
Can we catch a more specific exception? This will catch syntax errors too I think.

Reply:
Good point, now I catch `OrchestraException`